### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by PhilipFlyvholm** [PhilipFlyvholm/learnit-plus-plus#43] [Announcements]
- Reduced amount of epilepsy rave parties by 1000% on page change (No more flashes)
- Added theme toggling on the settings page
- New sidebar header
- Two new themes:
	- Hacker mode
	- Retro theme (OG LearnIT colors)
- Smoother page loading
- Fixed some bugs with dark mode
	- Login screen fix
	- Student forum fix
	- Increased contrast for notifications
- Technical changes:
	- New framework

<hr /><em>This discussion was created from the release <a href='https://github.com/PhilipFlyvholm/learnit-plus-plus/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by lucasfth** [comment]
It seems like the theme selector (the one LearnIT itself implemented which did not work) which was in general info has been removed.
But otherwise really great changes. Also, how was it tested that rave parties were reduced by 1000% haha.

**Reply by PhilipFlyvholm** [reply]
Oh wow great timing... And they still have the option to change your student number, which makes sense... We could maybe re-add a section for all the L++ settings.

**Reply by PhilipFlyvholm** [reply]
>  Also, how was it tested that rave parties were reduced by 1000% haha.

Waaay too much time used to test on firefox on windows
